adapters/Tapjoy/TapjoyAdapter/GADMAdapterTapjoy.m

```diff
@@ -125,16 +127,14 @@ - (void)requestDidSucceed:(nonnull TJPlacement *)placement {
   // If the placement's content is not available at this time, then the request is considered a
   // failure.
   if (!placement.contentAvailable) {
-    NSError *loadError =
-        GADMAdapterTapjoyErrorWithCodeAndDescription(kGADErrorNoFill, @"Ad not available.");
+    NSError *loadError = GADMAdapterTapjoyErrorWithCodeAndDescription(
+        GADMAdapterTapjoyErrorPlacementContentNotAvailable, @"Ad not available.");
     [_interstitialConnector adapter:self didFailAd:loadError];
   }
 }
 
 - (void)requestDidFail:(nonnull TJPlacement *)placement error:(nonnull NSError *)error {
-  NSError *adapterError =
-      GADMAdapterTapjoyErrorWithCodeAndDescription(kGADErrorNoFill, error.localizedDescription);
-  [_interstitialConnector adapter:self didFailAd:adapterError];
+  [_interstitialConnector adapter:self didFailAd:error];
```

Comment:
Any reason why this error now passes the NSError object through from our SDK instead of standardizing like the others?

Reply:
@melty710  The new error codes are for adapter's errors only.  Third Party SDK errors are not affected by these changes.